shapes/neurosciencegraph/datashapes/core/workflowengine/schema.json

```diff
@@ -0,0 +1,76 @@
+{
+  "@context": [
+    "https://incf.github.io/neuroshapes/contexts/schema.json",
+    {
+      "this": "https://neuroshapes.org/dash/softwareagent/shapes/"
+    }
+  ],
+  "@id": "https://neuroshapes.org/dash/workflowengine",
+  "@type": "nxv:Schema",
+  "imports": [
+    "https://provshapes.org/commons/agent"
```

Comment:
I think the workflow engine should extend [https://neuroshapes.org/dash/softwareagent](https://github.com/INCF/neuroshapes/blob/master/shapes/neurosciencegraph/datashapes/core/softwareagent/schema.json)